ui/src/app/pages/admin/batch-sync-user-access.tsx

```diff
@@ -11,7 +11,7 @@ import { SpinnerOverlay } from 'app/components/spinners';
 import { WithSpinnerOverlayProps } from 'app/components/with-spinner-overlay';
 import { userAdminApi } from 'app/services/swagger-fetch-clients';
 import { reactStyles } from 'app/utils';
-export const AdminUserAccess = (spinnerProps: WithSpinnerOverlayProps) => {
```

Comment:
renamed so as not to confuse with the all-users AdminTable page and the individual-user AdminUserProfile page.  This represents an admin action to perform on users.